README.md

```diff
@@ -1,10 +1,12 @@
 # Awesome Mechanical Keyboard [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
 ## Keyboards
-|  Name | Available Fieles | Tags | Image |
+|  Name | Available Files | Tags | Image |
 | --- | --- | --- | --- |
 | [Atreus](https://github.com/technomancy/atreus) | pcb, case | 40% | <img src="https://github.com/technomancy/atreus/raw/master/atreus.jpg" width="200px"/> |
+| [Chimera Ergo/Ergo Mini/Ergo 42](https://github.com/GlenPickle/Chimera) | pcb | split, wireless, vertically staggered | <img src="https://i.imgur.com/AA6ycMQ.jpg" width="200px"> <img src="https://i.imgur.com/LAny3ue.jpg" width="200px"> <img src="https://i.imgur.com/wfTsIKV.jpg" width="200px"> |
```

Comment:
@opcon can you remove two images? Every row should have one images.
BTW thanks for the new content

Reply:
There are three separate (but similar) keyboards included in that row, so there's an image showing each one. Would you prefer that I split it up into 3 separate rows, or only have an image of one of the keyboards?

And thanks for starting the list in the first place!

Reply:
Ah, now i saw it. It's fine thank you for your contribution.